actions/inventory.go

```diff
@@ -187,7 +190,7 @@ func Collect(ctx context.Context, device *common.Device, collectors *Collectors,
 
 // Drives executes drive collectors and merges the data into device.[]*Drive
 // nolint:gocyclo // TODO(joel) if theres more conditionals to be added in here, the method is to be split up.
-func Drives(ctx context.Context, drives []*common.Drive, collectors []DriveCollector) error {
+func Drives(ctx context.Context, device *common.Device, collectors []DriveCollector) error {
```

Comment:
This is cleaner than what I had in mind - passing a pointer to the drive slice, I had to check what the code would look like, and it wasn't pretty https://go.dev/play/p/lw5RB8lXdfR